### ISSUE TITLE
Adapt cartridge cli to new release policy

### ISSUE THREAD

**Issue by olegrok** [tarantool/cartridge-cli#619]
https://github.com/tarantool/cartridge-cli/blob/e9e99f68ccd90b5ad1cc0c435658c1add3a5c2e5/cli/common/tarantool.go#L21

This one leads to "⨯ Failed to get Tarantool context: Failed to get Tarantool version: Failed to match Tarantool version"

**Comment by rosik** [comment]
Another architectural problem: `cartridge pack --docker` tries to install tarantool inside the container. He does it using:

```
Step 3/5 : RUN curl -L https://tarantool.io/installer.sh | VER=2.10 bash
```

According to the new policy, there's no such thing as "VER=2.10", there's only "series-2".

Also, `cartridge pack` prescribes deb/rpm dependencies, which should also be revised.

**Comment by Totktonada** [comment]
And I would highlight that now we have only pre-relase (2.10.0-beta1) of the series-2. There is no any release. Releases and pre-releases are live in the separate repositories.

**Comment by rosik** [comment]
The version parsing was singled out in #621 and fixed in #622. The issue remains devoted to the management of repos and deps.
